Metalama.Framework/Aspects/ContractAspect.RedirectToProxyParameterAnnotation.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) SharpCrafters s.r.o. See the LICENSE.md file in the root directory of this repository root for details.
+
+using Metalama.Framework.Code;
+using System;
+using System.ComponentModel;
+
+namespace Metalama.Framework.Aspects;
+
+public abstract partial class ContractAspect
+{
+    /// <summary>
+    /// This class supports Metalama framework infrastructure and should not be used directly by user code.
+    /// </summary>
+    [CompileTime]
+    [EditorBrowsable( EditorBrowsableState.Never )]
+    public sealed class RedirectToProxyParameterAnnotation : IAnnotation<IFieldOrPropertyOrIndexer>, IAnnotation<IParameter>
```

Comment:
The class should be internal. There should be public static method instead:

```public static void RedirectContracts( IFieldOrPropertyOrIndexer sourceTarget, IParameter targetParameter )```

I don't think the redirection from `IParameter` is used.